deployment/terraform/sfn.tf

```diff
@@ -0,0 +1,47 @@
+#
+# Step Function Resources
+#
+data "template_file" "app_cli_state_machine" {
+  template = "${file("step-functions/app_cli.json")}"
+
+  vars = {
+    ecs_cluster_arn         = "${aws_ecs_cluster.app.arn}"
+    security_group_id       = "${aws_security_group.app.id}"
+    subnet_ids              = "${jsonencode(module.vpc.private_subnet_ids)}"
+    ecs_task_definition_arn = "${aws_ecs_task_definition.app_cli.arn}"
+    ecs_container_name      = "django"
+    timeout_seconds         = "${var.app_cli_state_machine_timeout_seconds}"
+    lambda_function_arn     = "${aws_lambda_function.alert_sfn_failures.arn}"
+  }
+}
+
+resource "aws_sfn_state_machine" "app_cli" {
+  name     = "stateMachine${var.environment}AppCLI"
+  role_arn = "${aws_iam_role.step_functions_service_role.arn}"
+
+  definition = "${data.template_file.app_cli_state_machine.rendered}"
+}
```

Comment:
I designed this to be something that we could reuse, independent of the rule that invokes `check_api_limits`. We can invoke this State Machine with whatever command we want. This means we could easily add more scheduled management commands.

Reply:
Good idea! This feels like the right abstraction.